mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import {Option, OptionType} from "src/component/select/option/Option";
+import styles from "src/component/select/Select.module.scss";
+
+
+/**
+ * Select props
+ */
+export interface SelectProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Select`s value
+   */
+  value: string;
+  /**
+   * Select's name
+   */
+  name: string;
+  /**
+   * Options list
+   */
+  options: OptionType[];
+  /**
+   * Callback triggered onChange select value
+   */
+  onChange: (value: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
now your interface is a lie. Did you check ant design select component? value is not an event

Reply:
in art design use handler 
const onChange = (value: string) => {
  console.log(`selected ${value}`);
};
but if i use it throw errow:
Type '(value: string) => void' is not assignable to type 'ChangeEventHandler<HTMLSelectElement>'.
  Types of parameters 'value' and 'event' are incompatible.
    Type 'ChangeEvent<HTMLSelectElement>' is not assignable to type 'string'.

we need use in props of handler e:React.ChangeEvent<HTMLSelectElement>

Reply:
done